dot_graph.py

```diff
@@ -11,7 +11,9 @@
 
 import sys
 import os
-import source
+from typing import TypeVar, Mapping
+import cxopen
+from source import Expr, ExprNum, ExprOp, ExprVar, Function, NodeBasic, NodeCall, NodeCond, NodeEmpty, Operator, Update, convert_function
```

Comment:
I think a lot things use `source.` in this file, so I don't think you meant that either